dashboard/lib/logic/links.dart

```diff
@@ -24,25 +25,36 @@ List<CocoonLink> createCocoonLinks(BuildContext context) {
       action: () => Navigator.pushReplacementNamed(context, BuildDashboardPage.routeName),
     ),
     CocoonLink(
-      name: 'Framework Benchmarks',
-      icon: const Icon(Icons.show_chart),
-      action: () => launch('https://flutter-flutter-perf.skia.org/'),
-    ),
+        name: 'Framework Benchmarks',
+        icon: const Icon(Icons.show_chart),
+        action: () {
+          launch('https://flutter-flutter-perf.skia.org/');
+          const MethodChannel('plugins.flutter.io/url_launcher')
+              .invokeMethod<void>('launch', 'https://flutter-flutter-perf.skia.org/');
+        }),
     CocoonLink(
-      name: 'Engine Benchmarks',
-      icon: const Icon(Icons.show_chart),
-      action: () => launch('https://flutter-engine-perf.skia.org/'),
-    ),
+        name: 'Engine Benchmarks',
+        icon: const Icon(Icons.show_chart),
+        action: () {
+          launch('https://flutter-engine-perf.skia.org/');
+          const MethodChannel('plugins.flutter.io/url_launcher')
+              .invokeMethod<void>('launch', 'https://flutter-engine-perf.skia.org/');
+        }),
     CocoonLink(
-      name: 'Repository',
-      icon: const Icon(Icons.info_outline),
-      action: () => launch('/repository.html'),
-    ),
+        name: 'Repository',
+        icon: const Icon(Icons.info_outline),
+        action: () {
+          launch('/repository.html');
+          const MethodChannel('plugins.flutter.io/url_launcher').invokeMethod<void>('launch', '/repository.html');
```

Comment:
Thanks a lot for all the comments! Yeah this the part I am mostly concerned. Because after migrating to null safety, if I keep it as it was with only the launch statement, it does not get captured by `setMockMethodCallHandler`and will fail a ton of tests. Would be great if I can keep launch as it as and still pass all the tests, but I wasn't successful in doing that.